planter/Dockerfile

```diff
@@ -61,4 +61,6 @@ RUN INSTALLER="bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh"; \
 RUN touch /etc/passwd && chmod a+rw /etc/passwd
 
 COPY ./entrypoint.sh /
+RUN chmod +x ./entrypoint.sh
```

Comment:
I think you need `+rx`.

Reply:
```
chmod: cannot access './entrypoint.sh': No such file or directory
The command '/bin/bash -c chmod +x ./entrypoint.sh' returned a non-zero code: 1
```
ah, the real culprit is that copy command